target/linux/ath79/dts/ar9344_araknis_an-300-ap-i-n.dts

```diff
@@ -87,11 +87,10 @@
 	ath9k: wifi@0,0,0 {
 		compatible = "pci168c,0030";
 		reg = <0x0 0 0 0 0>;
-		nvmem-cells = <&macaddr_art_0>;
-		nvmem-cell-names = "mac-address";
+		nvmem-cells = <&macaddr_art_0>, <calibration_ath9k_5000>;
+		nvmem-cell-names = "mac-address", "calibration";
 		mac-address-increment = <1>;
 		qca,disable-5ghz;
```

Comment:
`qca,disable-5ghz;` and `qca,disable-2ghz;` seem to have been abandoned. https://github.com/openwrt/openwrt/commit/4c8dd973ef8e76f9fc622f0baab981769c89b601
Maybe you can replace them with `ieee80211-freq-limit = <2400000 2500000>;` `ieee80211-freq-limit = <5000000 6000000>;` in this PR.

Reply:
Thanks, I think I will add a separate PR fixing that.

Reply:
https://github.com/openwrt/openwrt/pull/11520